proarc-common/src/main/java/cz/cas/lib/proarc/common/catalog/AlephXServer.java

```diff
@@ -17,8 +17,19 @@
 package cz.cas.lib.proarc.common.catalog;
 
 import cz.cas.lib.proarc.common.config.CatalogConfiguration;
+import cz.cas.lib.proarc.common.config.CatalogQueryField;
 import cz.cas.lib.proarc.common.mods.ModsUtils;
 import cz.cas.lib.proarc.common.xml.Transformers;
+import org.w3c.dom.Element;
+
+import javax.xml.bind.JAXB;
```

Comment:
Kvůli těm importům to nehodlám dál blokovat, ale pokud někdo začne používat ještě řazení podle standardně nastavené v eclipse, tak z toho bude vznikat hrůza.
Na googlu jsem našel, že lze v Idea vyjmenovat jen root folders (`import java.*`) v daném pořadí, což není tak hrozné. Tolik jich zase nepoužíváme (cz, com, java, javax, org).